sources/osg/State.js

```diff
@@ -457,31 +468,25 @@ State.prototype = MACROUTILS.objectLibraryClass( MACROUTILS.objectInherit( Objec
         }
     },
 
-    applyTextureAttribute: function ( unit, attribute ) {
+    applyTextureAttribute: function ( unit, attribute, force ) {
```

Comment:
No use case.
Framebuffer like VAO, if texture not really binded (not cache) just before, it's not framebuffer associated.
That's the simple way to make sure we do bind it.